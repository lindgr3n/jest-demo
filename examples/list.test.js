import sortit from './list'

it('Should correct sort the list', () => {
    const result = sortit(['b','c','a'])
    expect(result).toEqual(['a','b','c'])
})

it('Should correct sort the list', () => {
    const result = sortit(['b','c','a','ðŸ‘»'])   // cmd + ctrl + space
    expect(result).toMatchSnapshot()
})