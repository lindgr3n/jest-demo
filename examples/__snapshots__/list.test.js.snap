// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should correct sort the list 1`] = `
Array [
  "a",
  "b",
  "c",
  "ðŸ‘»",
]
`;
